src/frontend/qt_sdl/main.cpp

```diff
@@ -1872,6 +1878,21 @@ void MainWindow::onChangeAudioSync(bool checked)
     Config::AudioSync = checked?1:0;
 }
 
+void MainWindow::onCheckForUpdates()
+{
+    emuThread->emuPause();
+    int updateCheck = Updater::checkForUpdates("dummy ver");
+    if (!updateCheck)
+    {
+        QMessageBox::information(this, "melonDS", "You are already on the latest version.");
+        emuThread->emuUnpause();
+    }
+    else 
+    {
+        Updater::installUpdate()
```

Comment:
Should have a confirmation before actually installing, maybe with a link to release notes if that's possible.

Reply:
will do.

Reply:
You can just link to the latest release page, i.e. https://github.com/Arisotura/melonDS/releases/latest.

Reply:
will do

Reply:
got the commit info, so done.
now for stable releases.